cypress/integration/accountDetail.test.ts

```diff
@@ -0,0 +1,16 @@
+context("Account detail test", () => {
+  beforeEach(() => {
+    const host = Cypress.env("DAPP_HOST") as string
+    cy.visit(`${host}#/`)
+    cy.waitForReact()
+    cy.wait(3000)
+  })
+
+  it("renders account detail after click account avatar", () => {
+    cy.get(".walletStatus").should("exist")
+    cy.get(".walletStatus>button").click()
+    cy.get(".accountDetail").should("exist")
+    cy.get(".buttonGroup>button").contains("Change Account").click()
```

Comment:
If these tests are passing then that's good. In the future try to move to directly targeting `data-testid` attributes. This will help in improving the robustness of E2E tests. Classes can become obfuscated, clash, and more and so can doing things like targeting elements.

As explained here: https://docs.cypress.io/guides/references/best-practices#How-It-Works